pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

```diff
@@ -181,15 +182,19 @@ private void validateQueries() {
     invalidQueries.forEach(_queryWeightMap::remove);
   }
 
-  public void registerColnameFieldType() { // create a map from colname to data type
+  public void registerColNameFieldType() { // create a map from col name to data type
```

Comment:
Not related to the changes in this PR but can we do the following cleanup ?
- Make this private. It is not used elsewhere.
- Move the javadoc to one line before.

Reply:
Sure. Done :)

Reply:
still shows public ?

Reply:
😅 
Done now!